sound/soc/sof/control.c

```diff
@@ -67,7 +67,7 @@ static inline u32 ipc_to_mixer(u32 value, u32 *volume_map, int size)
 
 static void snd_sof_refresh_control(struct snd_sof_control *scontrol)
 {
-	struct sof_ipc_ctrl_data *cdata = scontrol->control_data;
+	struct sof_ipc_ctrl_data *cdata = scontrol->ipc_control_data;
```

Comment:
`struct sof_ipc_ctrl_data` is still IPC3 ;)

Reply:
should they be IPC agnostic? what's the plan if yes?